src/test/lrsql/util/auth_test.clj

```diff
@@ -119,6 +119,14 @@
              :scopes         #{}}
       false {:request-method :delete
              :path-info      "xapi/activities/state"
+             :scopes         #{}}
+      ;; Admin Scope
+      ;; Not used for xAPI, only for Admin requests
+      true {:request-method :get
+            :path-info      "/admin/account"
```

Comment:
Huh I guess you're right. We'll it's fortunate that the function isn't particularly strict about the nature of the path info string (e.g. it only checks if `statements` is a suffix for certain scopes). I guess you could make it more strict, but there's a small possibility it could be backwards incompatible.